docs/CONTRIBUTING.md

```diff
@@ -0,0 +1,22 @@
+# Contributing!
+
+🎉 First off, thanks for taking the time to contribute! 🎉
+
+## Guidelines
+
+The following is a set of guidelines for contributing to this repository. Use your best judgment, and feel free to propose changes to this document in a pull request.
+## Recommendations
+
+- Create a [topic branch](git-scm.com/book/en/v2/Git-Branching-Branching-Workflows#_topic_branch) on your fork for your specific PR.
+- Consider using [conventionalcommits.org](https://www.conventionalcommits.org/en/v1.0.0/)'s rules for creating explicit and meaningful commit messages.
+- If it's your first time contributing to a project then you should look to the popular [first-contributions](https://github.com/firstcontributions/first-contributions) repository on GitHub. This will give you hands-on experience with the features of GitHub required to make a contribution. As always, feel free to join our [Discord](https://discord.com/invite/r6Mdz5dpFc) to ask any questions and clarify your understanding, we are more than happy to help!
```

Comment:
It might be better to give more specific advice since we know that this is a mono repo:

There are specifically 2 types of PRs now:
- Port Requests
	- We don't need to say much here since there will be a generic PR template that is autofilled but these PRs should follow a strict set of rules (e.g. branch name should be `feat/<name-kebab-case>`
- Other (improvements to docs, scripts, etc)
	- Mentioning the topic branches is good here and conventional commits etc